src/components/toast-container/toast-container.scss

```diff
@@ -1,15 +1,43 @@
 :host {
   position: fixed;
-  bottom: 40px;
-  left: 32px;
   display: flex;
   flex-direction: column;
   z-index: $zindex-toast;
   width: 456px;
+  
+  &.bottom-right {
+    bottom: 40px;
+    right: 32px
```

Comment:
as the value of 32px and 40px repeats thru a lot of selectors maybe you could create a scss variable on the top of the SCSS file to store them, like
 ```$vertical-margin: 40px; ```
```$horizontal-margin: 32px;```